src/Symfony/Component/EventDispatcher/LegacyEventDispatcherProxy.php

```diff
@@ -11,8 +11,6 @@
 
 namespace Symfony\Component\EventDispatcher;
 
-use Psr\EventDispatcher\StoppableEventInterface;
-use Symfony\Contracts\EventDispatcher\Event as ContractsEvent;
 use Symfony\Contracts\EventDispatcher\EventDispatcherInterface as ContractsEventDispatcherInterface;
```

Comment:
we can remove the `as` now?

Reply:
We shouldn't because there already is an `EventDispatcherInterface` in the same namespace as this class.

Reply:
The `use` takes over.